frontend/src/components/channels/ChannelDropdownMenu.tsx

```diff
@@ -22,14 +24,19 @@ import { Channel } from "src/types";
 type ChannelDropdownMenuProps = {
   alias: string;
   channel: Channel;
-  editChannel(channel: Channel): void;
 };
 
 export function ChannelDropdownMenu({
   alias,
   channel,
-  editChannel,
 }: ChannelDropdownMenuProps) {
+  const [dialog, setDialog] = React.useState<"close" | "routingFee" | null>(
+    null
+  );
+
+  const openCloseDialog = () => setDialog("close");
```

Comment:
Can you read the name of this function out loud and ask yourself does it make sense?